app/views/lookup_keys/_fields.html.erb

```diff
@@ -1,59 +1,61 @@
-        <%= checkbox_f(f, :avoid_duplicates, :disabled => (!f.object.supports_uniq? || !f.object.merge_overrides), :size => "col-md-8",
-                          :help_block => _("Should the matched result avoid duplicate values (only array type).")) %>
-    </div>
-
-    <legend><%= _("Override value for specific hosts") %></legend>
-    <%= textarea_f f, :path, :rows => :auto, :label => _("Order"), :size => "col-md-8", :value => f.object.path,
-                      :help_block => popover(_("The order in which values are resolved"),
-                                             _("The order in which matchers keys are processed, first match wins.<br> You may use multiple attributes as a matcher key, for example, an order of <code>host group, environment</code> would expect a matcher such as <code>hostgroup = \"web servers\", environment = production</code>"), :title => _("The order in which values are resolved")).html_safe
-    %>
 
-    <div class="children_fields lookup_values">
-      <%= new_child_fields_template(f, :lookup_values, {:partial => "lookup_keys/value", :form_builder_attrs => {:is_param => is_param}}) %>
-      <%= f.fields_for :lookup_values do |lookup_values| %>
-        <%= render 'lookup_keys/value', :f => lookup_values, :is_param => is_param %>
-      <% end %>
-      <%= add_child_link '+ ' + _("Add Matcher-Value"), :lookup_values, { :title => _('add a new matcher-value pair')} %>
-      <%= documentation_button('4.2.6SmartMatchers') %>
-    </div>
+    <%= param_type_selector(f, :onchange => 'keyTypeChange(this)') %>
+    <%= textarea_f f, :default_value, :value => f.object.default_value_before_type_cast, :size => "col-md-8",
+                   :disabled                 => (f.object.is_param && (!f.object.override || f.object.use_puppet_default)),
+                   :rows                     => :auto, :help_inline => popover('', _("Value to use when there is no match")) %>
+    <%= checkbox_f(f, :use_puppet_default, :label => _('Use Puppet default'), :size => "col-md-8",
+                   :help_inline                   => use_puppet_default_help,
+                   :onchange                      => 'toggleUsePuppetDefaultValue(this, "default_value")',
+                   :disabled                      => (f.object.is_param && !f.object.override)) if is_param %>
+    </br>
+  </fieldset>
+  <div class="matcher-parent" <%= "id=#{(f.object.key || 'new_lookup_keys').to_s.gsub(' ', '_')}_lookup_key_override_value" %> style=<%= "display:none;" if (f.object.is_param && !f.object.override) %>>
+    <fieldset>
+      <legend><%= _("Prioritize Attribute Order") %></legend>
+      <h6><%= popover("", _("The order in which matchers keys are processed, first match wins.<br> You may use multiple attributes as a matcher key, for example, an order of <code>host group, environment</code> would expect a matcher such as <code>hostgroup = \"web servers\", environment = production</code>"),
```

Comment:
---
(separate thread)

Usually the 'i' is just on fields, not section introductions.  It's also usually after the item, not before.

Perhaps move some of the popover content into the section introduction and remove the popover entirely here?